### DISCUSSION TITLE
Image preview strip not scrolling to the beginning?

### DISCUSSION THREAD

**Discussion by Silanda** [vladmandic/sdnext#715] [General]
Starting from when I updated today, the strip of images showing all the images generated in the last run doesn't scroll to display the first few images at the beginning as I press the left or right keys to move through the images. The large image changes however. 

Funnily enough, one of the reasons I moved to this fork is that the same thing happened to my Automatic1111 build when I manually updated it to use Torch 2.0 a couple of weeks ago. One thing I noticed when I updated today was that it updated the Gradio version. Could that be the cause?

Is anyone else having this problem?

**Comment by TigerFox57** [comment]
Yes, same thing here.

**Comment by vladmandic** [comment]
i've ported a pr from a1111 that resutls in this. some people like it, some dont. i guess i can make it selectable by user. can you open a feature request so i don't forget?
